BankClient/obj/Debug/net6.0/BankClient.csproj.CoreCompileInputs.cache

```diff
@@ -1 +1 @@
-a75175fcb388a32d5c14bdd5399235c8a100833b
```

Comment:
You should exclude files that are not related to the code but related to the build. Please `.gitingore` file to exclude unnecessary files

Reply:
Ok